src/wallet/mod.rs

```diff
@@ -722,8 +700,8 @@ where
     /// *repalce by fee* (RBF). If the transaction can be fee bumped then it returns a [`TxBuilder`]
     /// pre-populated with the inputs and outputs of the original transaction.
     ///
-    /// **NOTE**: if the original transaction was made with [`TxBuilder::set_single_recipient`],
-    /// the [`TxBuilder::maintain_single_recipient`] flag should be enabled to correctly reduce the
+    /// **NOTE**: if the original transaction was made with [`TxBuilder::drain_to`],
+    /// the [`TxBuilder::maintain_drain_to`] flag should be enabled to correctly reduce the
     /// only output's value in order to increase the fees.
```

Comment:
nit: "only" doesn't necessarily apply here anymore, right?

Reply:
Yes! Thanks. What about we just rename this to `bump_by_shrinking` to remove the verbal association  with `drain_to` because it may be useful without `drain_to` (it can sitll be mentioned in the docs).
Or thinking ahead to when `bump_fee` is just a tx builder method that adds all the inputs and outputs of the previous transaction (so you can bump multiple at once), we could have a `allow_shrinking(script_pubkey)` which would push the associated output into a list that should be reduced to pay for a fee bump before adding further inputs. I'll leave that for the next PR but that's what I am thinking.

Reply:
I really like `allow_shrinking()`, I think it's very self explanatory. In the future we can extend that to take a `Vec` of script pubkeys instead of a single one